CHANGELOG.md

```diff
@@ -19,6 +33,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Removed `libphonenumber-js` from main bundle. Reduced bundle size limit by 20%.
 - Internal: Use `@sentry/browser` instead of `raven` to track Sentry events
 - UI: New Document Upload screen (**Note:** *changes introduced with this UI update include possible breaking changes for integrators with custom translations or copy*)
+- UI: As part of work to add the QR code option for cross device secure link the UX has been updated for the copy link and SMS options
```

Comment:
This should be under Changed section in 5.7.0 (probably my bad from when I was resolving the merge conflict)